init.sh

```diff
@@ -3,3 +3,4 @@
 set -m
 sudo apt-get update
 sudo apt-get install libeccodes-dev --assume-yes
+pip install netCDF4 h5netcdf
```

Comment:
We can add these to `requirements.txt`instead? On Databricks there's a separate spot where you can configure pip installs of packages on the cluster, and would be good to keep the python dependencies separate there.

Reply:
Good call! I will change this.